DB/src/main/java/io/deephaven/db/tables/lang/DBLanguageFunctionGenerator.java

```diff
@@ -558,6 +560,118 @@ public static void main(String args[]) {
             append(buf, castFromObjFormatter, BinaryExpr.Operator.PLUS, c, Object.class);
         }
 
+        // Special casts for PyObject to primitive
+        buf.append("        public static int intPyCast(Object a) {\n");
```

Comment:
There is some weirdness here.  Let me try to summarize.
* Python 3 has numeric type changes vs Python 2.  In Python 3, float is double, and int is of unlimited size.
* Our code is frequently using getIntValue , instead of getLongValue.  I assume this is a python 2 relic.  I'm guessing that getIntValue is doing a cast to the int32 range.  The cast to the int32 range could potentially drop null values in the int64 range, which is closer to the python `int`.
* For float, NULL_DOUBLE makes sense for the null value in python.  For int, we could argue that the null value could be NULL_LONG , since that is the closest reasonable value... but this null value is in the middle of the int range.  Ugh.
* Now for more diceyness.  In our casts, we don't treat objects differently if they are float or int.  To properly handle special null values, you would need to call getDoubleValue on float and getLongValue on int.
This leaves us in a weird place, if we try to do any NULL_<TYPE> value handling.

Reply:
Maybe the handling of special values for python types is too hard, given difference in the type system.

Reply:
We discussed this and think that we can't properly handle the special values in a clean way in Python, so Python will use the `None` value as the null identifier.

Reply:
We noted that we may need to verify that the data accessors prefer to give python None over NULL_INT etc. Otherwise, agreed that None is best.